en-US/Design.xml

```diff
@@ -702,7 +702,7 @@ $ vi myFile.txt
 	 	<section id="refer-object-names">
 			<title>Referring to Object Names</title>
 			 <para>
-				Do not use object names as part of normative grammar. A sentence should be complete without the object name.
+				When using object names, write a sentence so that it would be complete without the object name.
```

Comment:
Instead of "it would be complete" how about "is complete"

Reply:
Fixed